lib/Mojolicious/Plugin/AssetPack/Store.pm

```diff
@@ -17,6 +17,7 @@ has _types => sub {
   $t->type(otf   => 'application/font-otf');
   $t->type(ttf   => 'application/font-ttf');
   $t->type(woff2 => 'application/font-woff2');
+  $t->type(map   => 'application/json');
```

Comment:
From what I read, this is not required. Serving the map as "application/octet-stream" is just fine.